message tab_CharacterShootPatternsTable 
{
	optional string tname = 1;
	repeated tab_CharacterShootPatterns tlist = 2;
}

message tab_CharacterShootPatterns
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 Type = 3;
	optional int32 LightBullet = 4;
	optional int32 HeavyBullet = 5;
	optional int32 Missile = 6;
	optional int32 Nightning = 7;
	optional int32 Fog = 8;
	optional int32 Wind = 9;
	optional int32 Archibald = 10;
}